go/vt/vtgate/executor.go

```diff
@@ -465,6 +465,11 @@ func (e *Executor) handleSet(ctx context.Context, safeSession *SafeSession, sql
 		if k.Scope == sqlparser.GlobalStr {
 			return &sqltypes.Result{}, vterrors.New(vtrpcpb.Code_INVALID_ARGUMENT, "unsupported in set: global")
 		}
+
+		if k.Scope == sqlparser.VitessMetadataStr {
```

Comment:
Use a `switch` here now that you have >1 case?

Reply:
Done